clientupdate/clientupdate.go

```diff
@@ -717,7 +717,7 @@ func (up *Updater) updateWindows() error {
 	}
 
 	if !winutil.IsCurrentProcessElevated() {
-		return errors.New("must be run as Administrator")
+		return errors.New("update must be run as Administrator; right-click cmd.exe, select 'Run as administrator' and try again")
```

Comment:
Well, this requires the person to find cmd.exe in Explorer, which is a lot of clicking (even if they know the path to it). I suggest perhaps:

* Windows+x then a
* Windows+r then type "cmd" then Control+Shift+Enter

There is also `runas /user:Administrator "tailscale update"`, but that requires you to know the Administrator password — and on the default set up of at least Windows 11, there isn't one. So `runas` would not work for everyone. The other 2 methods above I think do.

Reply:
If the user ended up running `tailscale.exe update` in non-admin console of some kind (that's how they ended up in this code path), they can probably find `cmd.exe` again?
On Windows 11 you don't even need to find it in Explorer, you can use the search bar and right click from there.

I can swap the message to mention shortcuts, if you think that's easier for most users though.

Reply:
Well, by default, tailscale.exe's parent directory is added to the `PATH` environment variable, so it is possible to find yourself running tailscale from the CLI without necessarily knowing where it is. It seems most likely that they ended up in `cmd` from Windows+r.

Maybe something like, "You can run cmd as Administrator by: ..." and then list all 3 possibilities? Up to you.

Reply:
Sounds good, turns out multiline error string gets printed nicely!